### ISSUE TITLE
Prepare demo-data

### ISSUE THREAD

**Issue by v1r7u** [deepnetworkgmbh/joseki#148]
Prepare audit data that can be used to demo the project capabilities:
- adding/removing Azure subscription or k8s-clusters
- changes in the same components state over the time

The data should be easily to apply (restore?) and should cover at least last month since the request date.

- [x] azsk
- [x] polaris
- [x] trivy

**Comment by yufufi** [comment]
We're looking for demo data that shows different capabilities of the product. Probably we're looking at creating a database with mock data that we expose through our public/demo URL.

@v1r7u to come up with a proposal on how to do this.

**Comment by yufufi** [comment]
There are two options here:
* Mock scanner outputs and feed it to the system
* A database with mock data
* Mock at REST API Level.

@v1r7u , @detaybey will discuss

**Comment by v1r7u** [comment]
I decided to go the first path: mock scanner outputs.
I prepared test-data for polaris and azsk, while trivy is expected to do _real_ scans for discovered images. 
These fake-scanners:
- reuse most of real scanner code-base
- instead of auditing real infrastructure, they take a pre-generated files
- this entire feature implemented in the way, that scan results gradually changes from bad outcome (score < 50) to almost perfect result (score > 95) and back
- fake-scanners can run daily with real joseki as the normal ones.

With this implementation, we should be able to provision a demo joseki instance, which runs 24/7 the entire year and it behaves like a real instance - generating some results every day.
